flyteplugins/go/tasks/plugins/k8s/ray/ray.go

```diff
@@ -400,19 +401,31 @@ func (plugin rayJobResourceHandler) GetTaskPhase(ctx context.Context, pluginCont
 		return pluginsCore.PhaseInfoUndefined, err
 	}
 
-	switch rayJob.Status.JobStatus {
-	case rayv1alpha1.JobStatusPending:
-		return pluginsCore.PhaseInfoInitializing(rayJob.Status.StartTime.Time, pluginsCore.DefaultPhaseVersion, "job is pending", info), nil
-	case rayv1alpha1.JobStatusFailed:
-		reason := fmt.Sprintf("Failed to create Ray job: %s", rayJob.Name)
+	// Kuberay creates a Ray cluster first, and then submits a Ray job to the cluster
+	switch rayJob.Status.JobDeploymentStatus {
+	case rayv1alpha1.JobDeploymentStatusInitializing:
+		return pluginsCore.PhaseInfoInitializing(time.Now(), pluginsCore.DefaultPhaseVersion, "cluster is creating", info), nil
```

Comment:
Would it make sense to use `rayJob.CreationTimestamp.Time` here?

Reply:
actually, `time` is not used in the [pluginsCore.PhaseInfoInitializing](https://github.com/flyteorg/flyte/blob/4717c8d0a1f511e1b0cf220f8321b2ff52cadb0e/flyteplugins/go/tasks/pluginmachinery/core/phase.go#L228-L232)

Reply:
So `time.Now()` here is unused?